packages/fx-core/resource/package.nls.json

```diff
@@ -388,18 +388,18 @@
   "core.createProjectQuestion.apiSpec.operation.placeholder.skipExisting": "Methods defined in manifest.json are not listed",
   "core.createProjectQuestion.apiSpec.multipleValidationErrors.message": "Incompatible OpenAPI description document. Check output panel for details.",
   "core.createProjectQuestion.apiSpec.multipleValidationErrors.vscode.message": "Incompatible OpenAPI description document. Check [output panel](command:fx-extension.showOutputChannel) for details.",
-  "core.createProjectQuestion.meArchitecture.title": "Architecture of search based message extension",
-  "core.createProjectQuestion.declarativeCopilot.title": "Create declarative copilot",
+  "core.createProjectQuestion.meArchitecture.title": "Architecture of Search Based Message Extension",
```

Comment:
Architecture of Search-Based Message Extension

Reply:
just checked the doc and the term is "search-based message extension"
![image](https://github.com/user-attachments/assets/5871e198-45bc-45a0-a658-6a786f9ea80b)